test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
```

Comment:
still not understanding why this is needed?  the build(assemble script) should create it.